products/lua.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Lua
+category: lang
+iconSlug: lua
+permalink: /lua
+versionCommand: |-
+  lua -v
+releaseImage:
+  https://www.lua.org/images/timeline.png
+releasePolicyLink: https://www.lua.org/versions.html
+changelogTemplate: |
+  https://www.lua.org/versions.html#{{"__RELEASE_CYCLE__"| split: " " | first}}/
+activeSupportColumn: true
+releaseDateColumn: true
+eolColumn: Support Status
+
+releases:
+-   releaseCycle: "5.4"
+    releaseDate: 2020-06-29
+    support: true
```

Comment:
What does "support" means for Lua ? Is there two phase of support (such as for https://endoflife.date/grafana for instance) ?

I see on https://deploy-preview-4433--endoflife-date.netlify.app/lua that when a Lua release is EOL it is also not supported, so if there is only one support phase all `support` field must be removed and `activeSupportColumn` set to `false` (or also removed, as `false` is the default value for this property).

Reply:
AFAIK, there is no 'security support' for Lua releases.

Reply:
Ok, so there is only a single support phase. Will drop the _Active support_ column and just keep the _Support Status_ column. I will also rename this column to just "Support".

Reply:
Done in 1a7ebd888a1f22bcd5b17431f9f4e3ae0adcb644.